lib/wsdl.js

```diff
@@ -1124,6 +1124,7 @@ WSDL.prototype._initializeOptions = function (options) {
   // Allow any request headers to keep passing through
   this.options.wsdl_headers = options.wsdl_headers;
   this.options.wsdl_options = options.wsdl_options;
+  if (options.httpClient) this.options.httpClient = options.httpClient;
```

Comment:
Could you please either use the "curly brace" form of the `if` condition? Personally I think it makes the code more readable and I hope that I get a `jscs` rule set in `soap` as soon as possible :+1: